docs/api.md

```diff
@@ -7,32 +7,40 @@ permalink: /docs/api/
 ##Compiling a template
 Assuming you have the dust compiler in your JS environment, you can compile a Dust template source file to JavaScript form. The following will compile this very simple template "Hello {name}!" to a JavaScript string that will also register it as a template named "intro".
 
-	var compiledTemplate = dust.compile("Hello {name}!", "intro");
+```
+var compiledTemplate = dust.compile("Hello {name}!", "intro");
```

Comment:
Definitely, I'm just fixing formatting in this one and will fix the content in the next PR